api/src/main/scala/hmda/api/http/InstitutionsHttpApi.scala

```diff
@@ -71,9 +69,7 @@ trait InstitutionsHttpApi extends InstitutionProtocol with ApiErrorProtocol with
               }
             case Failure(error) =>
               filingsActor ! Shutdown
-              log.error(error.getLocalizedMessage)
-              val errorResponse = ErrorResponse(500, "Internal server error", path)
-              complete(ToResponseMarshallable(StatusCodes.InternalServerError -> errorResponse))
+              fiveHundred(path, error)
```

Comment:
When I first read this I found this confusing. Not the fact that it's a function, which I like, but the nomenclature. I would expect something like `completeWithInternalError`, `returnInternalError` or similar